Library/Homebrew/cli/args.rb

```diff
@@ -10,7 +10,7 @@ class Args < OpenStruct
       # undefine tap to allow --tap argument
       undef tap
 
-      def initialize(argv = ARGV.freeze, set_default_args: false)
+      def initialize(argv = ARGV.dup.freeze, set_default_args: false)
```

Comment:
We do want to `.freeze` `ARGV` at some point (although I agree here isn't the best place).

Reply:
It's frozen now in `#parse` by default.